ci-operator/step-registry/hypershift/hostedcluster/destroy/hostedcluster/hypershift-hostedcluster-destroy-hostedcluster-ref.yaml

```diff
@@ -27,6 +57,9 @@ ref:
   - mount_path: /etc/hypershift-ci-jobs-awscreds
     name: hypershift-ci-jobs-awscreds
     namespace: test-credentials
+  - mount_path: /etc/hypershift-pool-powervs-credentials
```

Comment:
I am not sure if this mount make sense, because the credentials are mounted in the cluster profile.

Reply:
We don't have hypershift-powervs cluster profiles created so far, will use this chance to create it and use it in this workflow. Once we make the cluster profile ready, we can get rid of this test-credentials secret.

Reply:
I think its better to create the cluster profile. I can help you on merging those changes.